config/routes.rb

```diff
@@ -15,4 +13,10 @@
     end
   end
 
+  resources :labels do
+    member do
+      get :delete
```

Comment:
This should be use `delete` instead of `get`

When you define routes, you can use following verbs: `get`, `post`, `put` and `delete`.
Their usage is following:
* `get` for getting resource(s), typically `show` and `index` actions
* `post` for creating resource
* `put` for updating resource
* `delete` for destroying resource

[Rail's routing guide](https://guides.rubyonrails.org/routing.html#crud-verbs-and-actions) has perfect documentation for it with a lot of examples.

If you want to check what routes your application has, you can run `bundle exec rails routes`

Reply:
I need your help regarding this comment, since I tried to change it as you said but it didn't work, so I would love if you can explain this to me.